src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -175,6 +177,19 @@ public void handleFavourites() {
         }
     }
 
+    /**
+     * Opens the statistics window or focuses on it if it's already opened.
+     */
+    @FXML
+    public void handleStatistics() {
+        if (statisticsWindow != null) {
+            statisticsWindow.close();
+        }
```

Comment:
perhaps Javadoc can be more accurate? It seems that you are opening a new window instead of focusing on it if it is already open.

Reply:
Noted